### PR TITLE
adopter slider new logos

### PR DESCRIPTION
**Pull request by rakeshPRaghu**
This PR will add logos from https://github.com/openebs/openebs/blob/master/ADOPTERS.md to the logo slider

### CODE REVIEW

File: website/src/components/AdoptersSlider/index.tsx [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+      logo: "../images/logos/zeta-associates.png",
+      name: t("home.usedInProductionBy.zetaAssociates"),
+    },
+  ];
+
+  return (
+    <Carousel settings={logoSliderSettings} className={classes.logoCarousel}>
+      {adopters.map(({ logo, name }) => {
+        return (
+          <div>
```
Comment (Pallavi-PH) [inline website/src/components/AdoptersSlider/index.tsx:154]: Please add key here

Comment (rakeshPRaghu) [reply website/src/components/AdoptersSlider/index.tsx]: added

File: website/public/locale/en/translation.yaml [hunk 2]
```diff
@@ -161,11 +161,31 @@ home:
...
+    agnes: agnes
+    byteDance: ByteDance
+    clew: CLEW
+    cloudNative: CLOUD NATIVE
+    codewave: CodeWave
+    cort: CORT
+    dRating: D-Rating
+    disid: disid
+    exact: exact
+    hamravesh: hamravesh
```
Comment (Pallavi-PH) [inline website/public/locale/en/translation.yaml:173]:
```suggestion
    hamravesh: Hamravesh
```

File: website/public/locale/en/translation.yaml [hunk 3]
```diff
@@ -161,11 +161,31 @@ home:
...
+    idnt: IDNT
+    kpn: kpn
+    kubesphere: KUBESPHERE
+    plaidCloud: PlaidCloud
+    rentHopper: RENTHOPPER
+    SHARECARE: sharecare
+    stratus5: Stratur5
```
Comment (Pallavi-PH) [inline website/public/locale/en/translation.yaml:180]:
```suggestion
    stratus5: Stratus5
```

File: website/src/components/AdoptersSlider/index.tsx [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+}));
+
+const AdopterSlider: React.FC = () => {
+  const classes = useStyles();
+  const { t } = useTranslation();
+
+  var logoSliderSettings = {
+    dots: false,
+    autoplay: true,
+    autoplaySpeed: 0,
```
Comment (Pallavi-PH) [inline website/src/components/AdoptersSlider/index.tsx:19]: Why is it 0?

File: website/src/components/AdoptersSlider/index.tsx [hunk 5]
```diff
@@ -0,0 +1,163 @@
...
+    speed: 10000,
+    slidesToShow: 1,
+    slidesToScroll: 1,
+    variableWidth: true,
```
Comment (Pallavi-PH) [inline website/src/components/AdoptersSlider/index.tsx:23]: All the logo widths are same

**Review by satyapriyamishra222 — APPROVED** [review]

File: website/src/components/AdoptersSlider/index.tsx (hunk 4 above)
Comment (rakeshPRaghu) [reply website/src/components/AdoptersSlider/index.tsx:19]: it is given to achive the smooth scroll effect on slider, by giving `autoplaySpeed:0` there wont be any delay on scroll

File: website/src/components/AdoptersSlider/index.tsx (hunk 5 above)
Comment (rakeshPRaghu) [reply website/src/components/AdoptersSlider/index.tsx:23]: it is given to achive the smooth scroll effect on slider, by giving `variableWidth:true` we can show any no of slides without worrying about the overlap of slides

Comment (Pallavi-PH) [reply website/src/components/AdoptersSlider/index.tsx:23]: Ok

**Review by Pallavi-PH — APPROVED** [review]

**Review by AVRahul — APPROVED** [review]

**Pull request merged**